src/main/java/org/generationcp/middleware/service/api/study/StudyGermplasmListService.java

```diff
@@ -1,7 +1,10 @@
 
 package org.generationcp.middleware.service.api.study;
 
+import org.generationcp.middleware.pojos.dms.StockModel;
+
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 public interface StudyGermplasmListService {
```

Comment:
We should rename to `StudyGermplasmService` and `StudyGermplasmServiceImpl`. We should stop the concept of working with germplasm "lists".

Reply:
Also rename to:
getGermplasmListFromPlots >> getGermplasmFromPlots
deleteStudyGermplasmList >> deleteStudyGermplasm
countStudyGermplasmList >> countStudyGermplasm
saveStudyGermplasmList >> saveStudyGermplasmList